CITATION.cff

```diff
@@ -0,0 +1,75 @@
+authors:
+  -
+    affiliation: "Technische Universität München"
+    family-names: Krusche
+    given-names: Stephan
+cff-version: "1.2.0"
+keywords:
+  - learning
+  - university
+  - feedback
+  - autograding
+  - assessment
+  - lti
+  - lms
+  - exercises
+  - programming-exercises
+  - interactive-learning
+  - artemis
+  - double-blind
+  - text-exercises
+  - modeling-exercises
+  - apollon
+  - exam-mode
+  - automatic-feedback
+license: MIT
+version: 5.1.3
+message: "If you want to reference ArTEMis in your work, please cite this paper."
+repository-code: "https://github.com/ls1intum/Artemis/"
+title: "ArTEMiS"
+preferred-citation:
+  authors:
+    -
+      affiliation: "Technische Universität München"
+      family-names: Krusche
+      given-names: Stephan
+    -
+      affiliation: "Technische Universität München"
+      family-names: Seitz
+      given-names: Andreas
+  title: "ArTEMiS: An Automatic Assessment Management System for Interactive Learning"
+  doi: 10.1145/3159450.3159602
+  year: 2018
+  isbn: "9781450351034"
+  publisher:
+    name: "Association for Computing Machinery"
+  keywords:
+    - "Automated Assessment"
```

Comment:
All of these are lowercase in the original, keeping it that way would make sense

Reply:
Removed them now anyway